pkg/query-service/model/v3/v3.go

```diff
@@ -187,6 +187,15 @@ const (
 	TagTypeResource TagType = "resource"
 )
 
+func (q TagType) Validate() error {
+	switch q {
+	case TagTypeColumn, TagTypeTag, TagTypeResource:
```

Comment:
@nityanandagohain We don't have TagTypeColumn as it's replaced by isColumn param.

Reply:
Sure.

Reply:
made the change